src/components/organisms/news/news.scss

```
@@ -0,0 +1,88 @@
+@use "../../variables" as *;
+
+.news-and-events-title {
+  & > h2 {
+    display: flex;
+    align-items: center;
+    font-size: $fontsize-l;
+    font-family: $fontfamily-mincho;
+    margin-bottom: 32px;
+
+    @media (min-width: $breakpoint-md) {
+      font-size: $fontsize-xxl;
+    }
+
+    .caption {
+      font-size: $fontsize-m;
+      font-family: $fontfamily-gothic;
+      font-weight: 400;
+      margin-left: 16px;
+    }
+  }
+}
+
+.news-contents {
+  list-style: square;
+  padding-left: 0px;
+  list-style: none;
+  background-color: $white;
+  margin-bottom: 0;
+
+  .news-content-link {
+    .news-content {
+      margin-top: -1px;
+      border: solid $gray_gainsboro 1px;
+      transition-timing-function: ease;
+      transition: 0.2s;
+
+      .news-date {
+        color: $gray_nobel;
+        font-size: $fontsize-xs;
+        margin-bottom: 0;
+        margin: 20px auto 0 20px;
+        line-height: 12px;

```

```suggestion
        line-height: 1;
```
こうすると、フォントサイズと同じ値と解釈されます。
font-sizeが変わってもそれに応じた高さに対応できるので、この書き方が推奨されています。
[mdn](https://developer.mozilla.org/ja/docs/Web/CSS/line-height)